### PR TITLE
Fix UI : Added check for airflow  while creating TestSuite

### LINKED ISSUE #7394: TestSuite deployment should check airflow api status
**Affected module**
UI

**Describe the bug**
Deploy is allowed to go through even when the airflow is not up and running
image

**To Reproduce**

Screenshots or steps to reproduce

**Expected behavior**

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Issue :-  Closes #7394 
-  Added Airflow check while adding TestSuite ingestion

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/TestSuiteIngestion.tsx [hunk 1]
```diff
@@ -175,6 +177,29 @@ const TestSuiteIngestion: React.FC<TestSuiteIngestionProps> = ({
...
+          } else {
+            setIsAirflowRunning(false);
+            reject();
+          }
+        })
+        .catch(() => {
+          setIsAirflowRunning(false);
+          reject();
+        });
+    });
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/TestSuiteIngestion.tsx:196]:
```suggestion
      checkAirflowStatus()
        .then((res) => {
          if (res.status === 200) {
            setIsAirflowRunning(true);
          } else {
            setIsAirflowRunning(false);
          }
        })
        .catch(() => {
          setIsAirflowRunning(false);

        });
```

File: openmetadata-ui/src/main/resources/ui/src/components/TestSuitePipelineTab/TestSuitePipelineTab.component.tsx [hunk 2]
```diff
@@ -263,11 +266,36 @@ const TestSuitePipelineTab = () => {
...
+        .then((res) => {
+          if (res.status === 200) {
+            resolve();
+          } else {
+            reject();
+          }
+        })
+        .catch(() => reject());
+    });
+  };
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/TestSuitePipelineTab/TestSuitePipelineTab.component.tsx:281]: It's too repetitive please make single function to check status

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/TestSuitePipelineTab/TestSuitePipelineTab.component.tsx]: @chirag-madlani  changes updated.

File: openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/TestSuiteIngestion.tsx (hunk 1 above)
Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/TestSuiteIngestion.tsx]: @chirag-madlani  changes updated.

**Review by chirag-madlani — APPROVED** [review]

**Review by ShaileshParmar11 — APPROVED** [review]

**Comment by ShaileshParmar11** [comment]
Merge this once meven passed

File: openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/TestSuiteIngestion.tsx [hunk 3]
```diff
@@ -188,15 +208,16 @@ const TestSuiteIngestion: React.FC<TestSuiteIngestionProps> = ({
...
             handleDeployClick={handleDeployClick}
             handleViewServiceClick={handleViewTestSuiteClick}
+            isAirflowSetup={isAirflowRunning}
             name={`${testSuite?.name}_${PipelineType.TestSuite}`}
             showDeployButton={showDeployButton}
             showIngestionButton={false}
             state={FormSubmitType.ADD}
             successMessage={getSuccessMessage}
             viewServiceText="View Test Suite"
+            onCheckAirflowStatus={handleAirflowStatusCheck}
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/TestSuiteIngestion.tsx:220]: Do we need this prop as we are passing the `isAirflowSetup={isAirflowRunning}` prop?

Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/TestSuiteIngestion.tsx:220]: yes we need these props, we have one button to recheck the status

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**